src/documents/views.py

```diff
@@ -666,3 +670,35 @@ def post(self, request, format=None):
             )
 
             return response
+
+
+class RemoteVersionView(GenericAPIView):
+    def get(self, request, format=None):
+        remote_version = "0.0.0"
+        is_greater = False
+        if settings.ENABLE_UPDATE_CHECK:
+            try:
+                with urllib.request.urlopen(
+                    "https://raw.githubusercontent.com/paperless-ngx"
+                    + "/paperless-ngx/main/src/paperless/version.py",
+                ) as response:
+                    remote = response.read().decode("utf-8")
+                match = re.search("(\\d+, \\d+, \\d+)", remote)
```

Comment:
I don't think this regex will match.  `\\` will match the literal `\`.

Reply:
It was probably a complaint about an invalid escape sequence.  Making it a raw string with fix that: `r"(\d+, \d+, \d+)"`

Reply:
Ah, thanks for clarifying. Yes that was exactly the issue and python syntax is weird =)
Fixed!